### DISCUSSION TITLE
Does FL103 (and other RS friction) have forced_rupture_time option for nucleation?

### DISCUSSION THREAD

**Discussion by jwjeremy** [SeisSol/SeisSol#1070] [Q&A] [answered]
There is force time rupture for linear-slip-weakening friction. Does rate-and-state friction, like FL103, have the option to use force-time-rupture as a nucleation option? I didn't see the options in the documentation. Thanks!

**Comment by sebwolf-de** [comment, marked as answer]
No, it is not supported.

**Reply by AliceGabriel** [reply]
Thanks. @jwjeremy maybe interested in implementing this feature.

**Reply by jwjeremy** [reply]
Thanks, Sebastian! Yes, I would like to learn how to help implement the feature!
